### PR TITLE
docs(package): added guidelines for release statuses

### PR DESCRIPTION
**Pull request by jgroth**
Added some guidelines on how consumers should handle the different release statuses, it will be visible at the top of the front page of the documentation. This is what made most sense to me, any feedback is welcome :smile:

### CODE REVIEW

**Comment by jgroth** [comment]
After writing this I found the official meaning of the tags according to TSDoc, but it's pretty similar to what I wrote :slightly_smiling_face: 
https://tsdoc.org/pages/tags/beta/

File: src/index.ts [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+ * following statuses
+ *
+ * - `@public` - released and ready to be used. The interface is stable and
+ * should not have any breaking changes without triggering a new major release.
+ * - `@beta` - the interface is quite stable and can be used by those willing
+ * to accept breaking changes. Breaking changes should be kept to a minimum and
+ * will not trigger a new major release. Feedback on the interfaces are welcome.
+ * - `@alpha` - still under development and is likely to have frequent breaking
+ * changes without notice.
+ * - `@deprecated` - Can be used for the time being, but will be removed in the
```
Comment (Befkadu1) [inline src/index.ts:15]:
```suggestion
 * - `@deprecated` - can be used for the time being, but will be removed in the
```

**Comment by Befkadu1** [comment]
Awesome 👏

File: src/index.ts (hunk 1 above)
Comment (adrianschmidt) [reply src/index.ts:15]: I think it's weird to start the first sentence with a lowercase letter and then end it with a period.

(It's even worse when single sentence descriptions are started with a lowercase letter and just ends without a period, and then some descriptions need more than one sentence and therefore need periods and uppercase letters 🤢)

I think the only way to be consistent, is to start sentences with uppercase letters, and end them with some kind of punctuation.

Comment (Befkadu1) [reply src/index.ts:15]: Yep,  I am not good in written English but I checked the consistency 😆

Comment (adrianschmidt) [reply src/index.ts:15]: I can fix, so we can get this one merged 👍

File: src/index.ts [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
+ * - `@public` - Released and ready to be used. The interface is stable and
  * should not have any breaking changes without triggering a new major release.
- * - `@beta` - the interface is quite stable and can be used by those willing
- * to accept breaking changes. Breaking changes should be kept to a minimum and
- * will not trigger a new major release. Feedback on the interfaces are welcome.
- * - `@alpha` - still under development and is likely to have frequent breaking
+ * - `@beta` - The interface is fairly stable and can be used by those willing
+ * to accept breaking changes without notive. Breaking changes may still happen
+ * without notice, and without triggering a new major release. Feedback on the
+ * interfaces are welcome.
```
Comment (adrianschmidt) [inline src/index.ts:13]: I made `@beta` sound a bit less attractive to use.

**Review by Befkadu1 — DISMISSED** [review]

File: src/index.ts (hunk 1 above)
Comment (adrianschmidt) [reply src/index.ts:16]: ⚡

File: src/index.ts [hunk 3]
```diff
@@ -2,6 +2,22 @@
  * This is the public API for Lime Web Components. It contains helper functions
  * and interfaces for all services available in the Lime CRM platform.
  *
+ * Everything that is exported from this package is marked with one of the
+ * following statuses
+ *
+ * - `@public` - Released and ready to be used. The interface is stable and
+ * should not have any breaking changes without triggering a new major release.
+ * - `@beta` - The interface is fairly stable and can be used by those willing
+ * to accept breaking changes without notive. Breaking changes may still happen
```
Comment (Befkadu1) [inline src/index.ts:11]:
```suggestion
 * to accept breaking changes without notice. Breaking changes may still happen
```

Comment (adrianschmidt) [reply src/index.ts:11]: Ah! Nice! I'll fix!

Comment (adrianschmidt) [reply src/index.ts]: Fixed and amended to the existing commit.

**Review by Befkadu1 — APPROVED** [review]

**Review by adrianschmidt — APPROVED** [review]

**Pull request merged**